src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectStyles = {};
+  const arrayStyles = sourceString.split(';');
```

Comment:
избегай слов array string object number когда объявляешь переменные
в аргументах функции это допустимо
```suggestion
  const stringParts = sourceString.split(';');
```

Reply:
fixed